common/TelemetryEvents/ModifyWindowsOptionalFeaturesEvent.cs

```diff
@@ -0,0 +1,60 @@
+﻿// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+using System;
+using System.Diagnostics.Tracing;
+using DevHome.Telemetry;
+using Microsoft.Diagnostics.Telemetry;
+using Microsoft.Diagnostics.Telemetry.Internal;
+using static DevHome.Common.Scripts.ModifyWindowsOptionalFeatures;
+
+namespace DevHome.Common.TelemetryEvents;
+
+[EventData]
+public class ModifyWindowsOptionalFeaturesEvent : EventBase
+{
+    public override PartA_PrivTags PartA_PrivTags => PrivTags.ProductAndServicePerformance;
+
+    public string FeaturesString
+    {
+        get;
+    }
+
+    public string ExitCode
+    {
+        get;
+    }
+
+    public long DurationMs
+    {
+        get;
+    }
+
+    private readonly string[] exitCodeStrings =
+[
+        "Success",
+        "NoChange",
+        "Failure",
+        "Cancelled",
+    ];
+
+    public ModifyWindowsOptionalFeaturesEvent(string featureString, ExitCode result, long durationMs)
+    {
+        FeaturesString = featureString;
+        ExitCode = exitCodeStrings[(int)result];

```

Comment:
`ExitCode` and `ExitCodeStrings` needs to stay in sync.  Is it possible to move the enum to a common area?